### ISSUE TITLE
[CAIA intake] Content and IA Request for Health Conditions domain from mhv-medical-records

### ISSUE THREAD

**Issue by alexiawunder** [department-of-veterans-affairs/va.gov-team#55310]
## What does your team need support for? Check all that apply.
- [X] Launching one or more unauthenticated pages, online forms, apps, or tools on VA.gov
- [ ] Making revisions to one or more existing VA.gov pages
- [ ] Something else 
 
Note: Based on conversations with Laura and Kristin, this is a content-only ticket. Kristin will combine all of the domains into one IA ticket, but all of the domains will be broken out into individual content tickets. All of medical records will go through the regular collab cycle ticket together.

## Will this new product be released incrementally (for instance 25% of users initially)? 
  - [X] Yes
  - [ ] No
 
**Note:** If you checked yes, we'll reach out to discuss details about the content in the react widget.
  
## When do you expect to launch your product to 100% of users?
October 2024
  
*Please provide an estimated date so our team can create other relevant tickets.*
  
## Supporting artifacts
*Please provide supporting artifacts as available.*

- Link to product outline: [Medical Records Product Outline](https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/products/health-care/digital-health-modernization/mhv-to-va.gov/medical-records)
- Link to or attach [user flow](https://www.sketch.com/s/0803544d-e78c-423a-a70e-7db01a9a31c7/a/uuid/BFA923E3-109D-4819-97BF-3E258AF1FCCF)
- Link to [Domain Brief](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/health-care/digital-health-modernization/mhv-to-va.gov/medical-records/data-domains/health-issues/health-issues-brief.md)
- Link to prototype or mockups:
  - [Health conditions list view ](https://www.sketch.com/s/f6118ebb-966d-4f83-afba-f9e987b65eac/a/uuid/39DBAE50-6487-4084-8BBD-8E41A286BEDB)
  - [Details view](https://www.sketch.com/s/f6118ebb-966d-4f83-afba-f9e987b65eac/a/uuid/E40475C0-B20C-4249-B52D-C4F083040FAB)
- Link to any epics/issues that may be helpful: N/A

## Will this work be going through the Collaboration Cycle?
- [X] Yes 
- [ ] No

## When does this work need to be done?
- Estimated launch date: ~June 2023
- Estimated staging review date:~June 2023
- Content and IA work needed by: We'd like to conduct usability testing with approved content at the end of March, 2023

## Do you plan to bring this to an upcoming content office hours session?
- [ ] Yes, benefit content office hours (Thursdays, 3:00 p.m. to 3:45 p.m. ET)
- [ ] Yes, health  content office hours (Thursdays, 11:00 a.m. to 11:30 a.m. ET)
- [ ] Yes, unauth office hours (Mondays, 11:00 a.m. to 11:30 a.m. ET)
- [ ] No, but I'd like to schedule time to talk about this request
- [X] No, let's work asynchronously and meet if needed

**Note:** If we think this work would benefit from a collaborative session with you, we may ask you to bring it to office hours or set up a separate time to meet.

## About your team

- Team name: mhv-medical-records
- OCTO-DE product owner: Lauren Alexanderson
- Product manager: Bryan Riley
- Designer: Alexia Wunder, Bobby Bailey
- FE engineer: Mike Moyer
- Product/team Slack channel: #mhv-medical-records

## Next steps

**Once you’ve submitted this ticket, please post a link to this issue in the [#sitewide-content-ia](https://dsva.slack.com/channels/sitewide-content-ia) Slack channel and tag Randi Hecht.**

If you also need engineering support from the Public Websites team, fill out their [intake request form](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=Public+Websites%2C+Sitewide+content&labels=vsa-public-websites%2C+vsa%2C+vsa-public-websites-intake%2C+sitewide-content%2C+needs-grooming&template=public-websites-intake.md&title=%3CType+of+Request%3E+from+%3CTeam%3E).
  
If you need a page/URL redirected, a URL changed or a vanity URL set up, please submit a [Redirect, URL change, or vanity URL request](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new?assignees=mnorthuis&labels=ia&template=redirect-request.md&title=Redirect+Request)

**Comment by laurwill** [comment]
Hi @alexiawunder and @BobbyBaileyRB ,
Updates for health conditions domain:
- Intro sentence and "What to know" in [Medical records working content draft](https://dvagov-my.sharepoint.com/:w:/g/personal/laura_willwerth_va_gov/Eacz1GNGqrdNn3b5gC1j0n4BjNyZ6LyafJCpEfebp-cPVg?e=f6e3Hy)
- Comments on mobile views of Sketch screens

Remaining questions:
- We're only showing "Active" conditions, but still need to confirm if these are always **diagnosed** conditions — is it possible that a condition will appear in this list, but the Veteran was never actually diagnosed with this condition? If so we'll need a note about this.

**Comment by laurwill** [comment]
@alexiawunder i'm closing this ancient ticket, since health conditions domain is long-launched :)
